docs/book/src/SUMMARY.md

```diff
@@ -10,7 +10,7 @@
 - [Examples](./examples/index.md)
   - [Counter](./examples/counter.md)
   - [Subcurrency](./examples/subcurrency.md)
-  - [FizzBuzz](./examples/fizzbuzz.md)
+  - [`FizzBuzz`](./examples/fizzbuzz.md)
```

Comment:
Why is only this one put in backticks?

Reply:
It's just the only one that isn't really a word